routes/deductions/deductions.spec.js.js

```diff
@@ -0,0 +1,3 @@
+const request = require('supertest')
+const cheerio = require('cheerio')
+const app = require('../../app.js')
```

Comment:
One test for a 200 response and one for the redirect would be good.

Reply:
done!